assets/src/stories-editor/plugins/context-menu-item.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Plugin for add remove page button to more settings.
+ */
+
+/**
+ * External dependencies
+ */
+import { ReactElement } from 'react';
+/**
+ * WordPress dependencies
+ */
+import { PluginBlockSettingsMenuItem } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { isPageBlock } from '../helpers';
+
+export const name = 'amp-story-page-remove';
+
+export const icon = 'trash';
+
+/**
+ * Renders remove page button.
+ *
+ * @return {ReactElement} The rendered trash menu item.
+ */
+const RemovePageSetting = () => {
+	const { removeBlock } = useDispatch( 'core/block-editor' );
+	const currentPageId = useSelect( ( select ) => select( 'amp/story' ).getCurrentPage(), [] );
+
+	const removePage = () => {
+		if ( ! isPageBlock( currentPageId ) ) {
```

Comment:
When will `getCurrentPage()` ever return something that isn't a page?

Reply:
Good question. I don't think there's ever such a case. So this callback could be simplified.

Could probably even use `useCallback` to get a memoized callback:

```js
const removePage = useCallback(
  () => removeBlock( currentPageId ),
  [ currentPageId, removeBlock ],
);
```

Reply:
I think I was being overly careful here. I have removed the `isPageBlock` check. 

Not sure on the benefit of `useCallback` but I have put in anyway. https://github.com/ampproject/amp-wp/pull/3538/commits/5a65afcebe406c461f7fc134877f6b101f5d08da